setup/advanced/ipv6.md

```diff
@@ -0,0 +1,44 @@
+# IPv6
+
+⚠️ IPv6 is an experimental Docker feature, no thorough test was done to ensure there is no IP/DNS leak with Gluetun configured with IPv6. Feel free to create an issue or pull request if you have some testing done and can confirm.
+
+## Setup
+
+1. On your Docker host, edit and create if needed `/etc/docker/daemon.json` with the following JSON key-value pairs:
+
+    ```json
+    {
+      "ipv6": true,
+      "fixed-cidr-v6": "fd00::/80",
+      "experimental": true,
+      "ip6tables": true
+    }
+    ```
+
+    [Reference: Docker documentation on IPv6](https://docs.docker.com/config/daemon/ipv6/#use-ipv6-for-the-default-bridge-network)
+
+    ⁉️ IP range change to unique local address following the note at button of the documentation.
+1. Restart the Docker daemon to reload its JSON configuration. Most Linux distributions use `sudo systemctl restart docker` to do this.
+1. Edit your Gluetun `docker-compose.yml` and add the `sysctls` section and modify `WIREGUARD_ADDRESSES` to have both an IPv4 and an IPv6 address:
```

Comment:
> and add the `sysctls` section

Are you sure we need both the modification to `/etc/docker/daemon.json` and the `sysctls` block? The comment you previously linked was mentioning to do either one or the other 🤔

Reply:
![](https://i.imgur.com/PgvCqVe.png)

Reply:
The picture is what I see when I don't have sysctls section

Reply:
Cool, leaving it as is then!